newfold-labs/wp-module-patterns
# Feature - CSS Section Dividers

### Summary
This PR introduces buttons to the inspector control for modifying the separator style of the top-level group block only.

### Changes
- Added buttons to inspector control UI
- Enabled style modification for the top-level group block separator

### Testing Instructions

- Select a top-level group block and open the Block Settings.
- Modify the separator style using the new buttons under Divider Style.
- Check if changes apply to the top-level group separator.

### Screenshot
<img width="1334" alt="Screenshot 2023-10-09 at 18 54 11" src="https://github.com/newfold-labs/wp-module-patterns/assets/2559840/65639ea4-8a7b-4739-9e3b-c111e1b70060">
